.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - master
+        release-please--branches--master--components--nvim-tree
```

Comment:
Shouldn't that be an element of the list?

Reply:
Right you are... I fail at YAML

#2621